### ISSUE TITLE
login/lobby: height on div wrapper is incorrect, possible reason why buttons getting cutoff

### ISSUE THREAD

**Issue by jpailthorpe** [engagementgamelab/at-stake-v3#60]
overall these layouts need to be readjusted based on new mockups in zep. (text sizing is off and such) but a quick fix for now might just be hiding stakeholders being shown and change the wrapper size. 
![height on div wrapper is incorrect main reason why button getting cutoff](https://cloud.githubusercontent.com/assets/10051544/24381914/09bdb4ee-1321-11e7-92d5-0ee950f720a7.png)

**Comment by jpailthorpe** [comment]
Still seeing this issue, but we are closer. I noticed that there were some stakeholder roles in the cms that weren't updated with shorter names. I did that, so there won't ever be a stakeholder name that is longer than 2 lines. 



I think all we need to do is set the #gamecontent container top padding to 20 instead of 40, and make it so it only displays 4 stakeholder cards, not 6. That way we end up with this: 

This is how it should look in the build with these two changes: 
![card selection correct image in code](https://cloud.githubusercontent.com/assets/10051544/24477669/e0de6c86-14a5-11e7-998a-cd2e8abfaa65.png)

Also check line height. It seems there is a global line height being set in body, which is 1.44 Changing this to 1 makes it closer to the mock in zep. 
![card selection correct image in code2](https://cloud.githubusercontent.com/assets/10051544/24478095/9a450a80-14a7-11e7-9c9e-48caaefb2402.png)
